cmd/release-controller/http.go

```diff
@@ -237,6 +237,8 @@ func (c *Controller) userInterfaceHandler() http.Handler {
 	mux.HandleFunc("/releasestream/{release}/release/{tag}/download", c.httpReleaseInfoDownload)
 	mux.HandleFunc("/releasestream/{release}/latest", c.httpReleaseLatest)
 	mux.HandleFunc("/releasestream/{release}/latest/download", c.httpReleaseLatestDownload)
+	mux.HandleFunc("/candidate/{release}/list", c.httpReleaseCandidateList)
+	mux.HandleFunc("/candidate/{release}", c.httpReleaseCandidate)
```

Comment:
If this is an API call, it should be at `/api/v1/releasestream/{release}/candidate`.  Only html pages are allowed at the root.